introduction.md

```diff
@@ -1,2 +1,7 @@
 Another awesome course by CodeSchool? Yes, it really is, you guys did it again.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?